crates/gosub_html5/src/tokenizer/test.rs

```diff
@@ -0,0 +1,72 @@
+use gosub_testing::testing::tokenizer::{self, FixtureFile};
+use lazy_static::lazy_static;
+use std::collections::HashSet;
+use test_case::test_case;
+
+#[cfg(test)]
+mod tests{
```

Comment:
I don't think an extra `tests` module is necessary here. You already made a module by adding the file. So, I guess rename the file to `tests.rs` and add `#[cfg(test)` over the module definition in tokenizer.rs (which is missing btw)